packages/chakra-admin/src/components/layout/sidebar/AccountBox.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC } from 'react'
+import { Avatar, Box, Flex, FlexProps, Icon, Text } from '@chakra-ui/react'
+import { FiLogOut } from 'react-icons/fi'
+import { LogoutButton } from '../../buttons/LogoutButton'
+import { useAuthUser } from '../../../core/auth/useAuthUser'
+
+type Props = {} & FlexProps
+
+export const AccountBox: FC<Props> = ({ ...props }) => {
+  const { initialized, user } = useAuthUser()
+
+  return (
+    <Flex
+      padding="13px 16px"
+      borderTop="1px solid #E9E9E8"
+      justifyContent="space-between"
+      alignItems="center"
+      bg="white"
+      {...props}
+    >
+      <Flex width="fit-content" alignItems="center">
+        <Box border="2px solid #DBDBDB" w="38x" h="38px" borderRadius="50%">
+          <Avatar src={user?.avatarUrl} maxW="34px" maxH="34px" bg="#DBDBDB" />
+        </Box>
+        <Box ml={4}>
+          <Text
+            color="gray.700"
+            fontWeight="600"
+            textTransform="capitalize"
+            lineHeight="17.64px"
+            letterSpacing="0.2px"
+            fontSize="14px"
+          >
+            {user?.roles[0].name}
+          </Text>
+          <Text textStyle="description" textTransform="capitalize">
```

Comment:
I noticed that the text goes in two lines, can we add the `noOfLines={1}` prop to the role name and username?